src/test/java/io/lettuce/core/pubsub/RedisPubSubAsyncCommandsImplUnitTests.java

```diff
@@ -198,4 +198,22 @@ void pubsubShardNumsub() throws ExecutionException, InterruptedException {
 
         assertNotEquals(capturedCommand.getValue(), dispachedMock);
     }
+
+    @Test
+    void ssubscribe() throws ExecutionException, InterruptedException {
+        String pattern = "channelPattern";
+        AsyncCommand dispachedMock = mock (AsyncCommand.class);
+        when(mockedConnection.dispatch((RedisCommand<String, String, Object>) any())).thenReturn(dispachedMock);
+
+        commands.ssubscribe(pattern).get();
+
+        ArgumentCaptor<AsyncCommand> capturedCommand = ArgumentCaptor.forClass(AsyncCommand.class);;
+        verify(mockedConnection).dispatch(capturedCommand.capture());
+
+        Assertions.assertEquals( SSUBSCRIBE, capturedCommand.getValue().getType());
```

Comment:
Please use AssertJ, also we do typically statically import methods. Also, mind the code formatting (space between `assertEquals(` and `SSUBSCRIBE`, double space in `"key<channelPattern>",  capturedCommand`).

Reply:
fixed formatting, i ll send another pr for assertj